src/domain/image/validation.ts

```diff
@@ -73,10 +73,16 @@ export const addImageSchema = Yup.object().shape(
       [ADD_IMAGE_FIELDS.IMAGE_FILE, ADD_IMAGE_FIELDS.URL],
       validateSelectedImage as any
     ),
-    [ADD_IMAGE_FIELDS.IMAGE_FILE]: Yup.mixed().when(
-      [ADD_IMAGE_FIELDS.SELECTED_IMAGE, ADD_IMAGE_FIELDS.URL],
-      validateFile as any
-    ),
+    [ADD_IMAGE_FIELDS.IMAGE_FILE]: Yup.mixed()
+      .test(
+        'fileNameLength',
+        'image.form.validation.fileNameTooLong',
```

Comment:
Could also be added to VALIDATION_MESSAGE_KEYS enum. All other validation error i18n paths are added there.